src/Client.cpp

```diff
@@ -34,6 +34,7 @@
 #include "Switches.h"
 
 TicksTime s_startTime, s_endTime;
+uint32_t g_warmup_usec;
```

Comment:
What is a reason to use usec instead of msec for this value. I see in code that in all places you use de facto msec.

Reply:
please move this variable to `class App` and use `g_pApp->`

Reply:
struct itimerval's units are in usec:
timer->it_interval.tv_usec
But Sockperf's constants are in millisec.